docs/sources/api-response-examples/github/graph_api_users_members.json

```diff
@@ -18,6 +18,20 @@
               "login": "fake2",
               "organizationVerifiedDomainEmails": []
             }
+          },
+          {
+            "node": {
+              "email": "",
```

Comment:
Without the fix, that was being serialized as a hash of `""` instead of just, nothing.